### PR TITLE
T1881

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: src/plugin/plugin-list.js [hunk 1]
```diff
@@ -63,6 +63,11 @@ define( [ "util/dragndrop", "util/lang", "editor/editor", "text!layouts/plugin-l
 
       _containerElement.appendChild( element );
     });
+
+    // Open the plugin-list editor right after butter is finished starting up
+    butter.listen( "ready", function(){
```
Comment (k88hudson) [inline src/plugin/plugin-list.js]: whitespace

File: src/ui/header.js [hunk 2]
```diff
@@ -18,14 +19,12 @@ define( [ "dialog/dialog", "util/lang", "text!layouts/header.html" ],
     _title = _rootElement.querySelector(".butter-name");
     _title.innerHTML = options.value( "title" ) || "Popcorn Maker";
 
-    _rootElement = document.body.insertBefore( _rootElement, document.body.firstChild );
+    _saveButton = _rootElement.querySelector( ".butter-header-save" );
+    _sourceButton = _rootElement.querySelector( ".butter-header-source" );
+    _shareButton = _rootElement.querySelector( ".butter-header-share" );
+    _authButton = _rootElement.querySelector( ".butter-header-auth" );
```
Comment (k88hudson) [inline src/ui/header.js:25]: hehe

File: src/ui/header.js [hunk 3]
```diff
@@ -174,6 +173,11 @@ define( [ "dialog/dialog", "util/lang", "text!layouts/header.html" ],
       });
     }
 
+    this.attachToDOM = function(){
```
Comment (k88hudson) [inline src/ui/header.js]: whitespace

File: src/ui/ui.js [hunk 4]
```diff
@@ -108,6 +109,11 @@ define( [ "core/eventmanager", "./toggler",
     };
 
     Object.defineProperties( this, {
+      enabled: {
+        get: function(){
```
Comment (k88hudson) [inline src/ui/ui.js]: whitespace

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d09fdecb370ab04/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/4f2a21657e44990/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/d09fdecb370ab04/output.txt

Total script time: 0.79 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/4f2a21657e44990/output.txt

Total script time: 2.00 mins

Server is running on http://pr695.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/76d18ec7381c67e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/76d18ec7381c67e/output.txt

Total script time: 0.79 mins

Server is running on http://pr695.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/bdafd5d4f626ecc/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/bdafd5d4f626ecc/output.txt

Total script time: 1.19 mins

Server is running on http://pr695.butyr.org/templates/test.html

File: css/tray.less [hunk 5]
```diff
@@ -38,7 +38,7 @@
 
   // This is the list of plugins in "my events"
   .butter-timeline-area {
-    padding: 0 150px 0 0;
+    padding: 0;
```
Comment (mjschranz) [inline css/tray.less]: Do we need the padding reset here? Seems to me like this probably could be omitted.

File: src/ui/header.js (hunk 2 above)
Comment (mjschranz) [inline src/ui/header.js:25]: I'm confused

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e04cec145e63655/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e04cec145e63655/output.txt

Total script time: 0.89 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/9b18f9fea89d17c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/9b18f9fea89d17c/output.txt

Total script time: 1.31 mins

Server is running on http://pr695.butyr.org/templates/test.html

File: src/ui/header.js (hunk 2 above)
Comment (secretrobotron) [inline src/ui/header.js:25]: Changed to classes instead of ids :).

**Pull request merged**